app/domain/authentication/authn_oidc/pkce_support_feature/data_objects/authenticator.rb

```diff
@@ -28,7 +48,8 @@ def initialize(
             redirect_uri: nil,
             name: nil,
             response_type: 'code',
-            provider_scope: nil
+            provider_scope: nil,
+            token_ttl: nil
```

Comment:
Instead of passing `nil` here, what about setting it to the default token TTL (`PT8M`).  That'll allow you to avoid the nil check below.

Reply:
One question on avoiding the nil check - could someone still explicitly pass in nil? Does that need to be caught?